client/my-sites/plugins/index.js

```diff
@@ -68,19 +68,17 @@ export default function() {
 			siteSelection,
 			navigation,
 			ifSimpleSiteThenRedirectTo( '/plugins' ),
-			pluginsController.plugins.bind( null, 'all' ),
+			pluginsController.plugins,
 			sites
 		);
 
-		[ 'active', 'inactive', 'updates' ].forEach( filter =>
-			page(
-				`/plugins/${ filter }/:site_id?`,
-				siteSelection,
-				navigation,
-				pluginsController.jetpackCanUpdate.bind( null, filter ),
-				pluginsController.plugins.bind( null, filter ),
```

Comment:
Are we sure we can drop the `bind`s?

Reply:
Oh, I only noticed https://github.com/Automattic/wp-calypso/pull/20019/files#diff-9da1be70340f8dced3af9a3b94f4374fR190 now. Nevermind. Shouldn't have started reviewing from the bottom 😄